### ISSUE TITLE
dynamic :saml_authenticatable

### ISSUE THREAD

**Issue by paulmdi** [apokalipto/devise_saml_authenticatable#115]
Hi!
Is there a way to toggle on/off the `:saml_authenticatable`, e.g. based on a setting of a user

thanks

**Comment by adamstegman** [comment]
Yes, but it's not part of this gem's API. Devise is statically configured, so you have to jump through some hoops to make it dynamic.

I would suggest using [our wiki page on multiple strategies](https://github.com/apokalipto/devise_saml_authenticatable/wiki/Supporting-multiple-authentication-strategies) as a starting point. You'll have to find some way of figuring out the correct route to redirect to from your landing page.
